src/legacy/core_plugins/timelion/server/routes/run.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-disable @typescript-eslint/no-var-requires */
+import Joi from 'joi';
+import Bluebird from 'bluebird';
+import _ from 'lodash';
+import { Legacy } from 'kibana';
+// @ts-ignore
+import chainRunnerFn from '../handlers/chain_runner.js';
+const timelionDefaults = require('../lib/get_namespaced_settings')();
+
+export interface TimelionRequestQuery {
+  payload?: {
```

Comment:
@TinaHeiligers I hope you can help me out here for a sec - isn't payload a mandatory property, but `extended` is optional?

Reply:
You are correct, payload is required. I didn't remember to change that after deleting the [`GET` route ](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/timelion/server/routes/run.js#L34). Having both a `GET` and a `POST` as methods on the same route meant we had to accept either a [`payload` for the `POST` or a `query` object for the `GET`]((https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/timelion/server/routes/run.js#L47). `GET` wasn't being used at all so I deleted it.